software-engineering-policies/FrontEnd/FrontEndPolicy.md

```diff
@@ -0,0 +1,45 @@
+# Front End Policy
+
+
+
+## Design System
+
+
+
+At the UKHO we have an [in house design system](https://github.com/UKHO/admiralty-design-system).
+
+
+
+The design system repository is public which means that anyone in the world can look at the repository code and commits. We are currently not taking any contributions from people outside the UKHO and offer no support to external developers. 
+
+
+
+For internal developers we offer support through the usage of Github issues. There are two types of issue template, bug and feature request, these are templates that will guide developers to provide us with enough information to work on the issue. 
+
+
+
+The design system is supported primarily by the UX Champion team, however they only have part of their time dedicated to support, therefore if requests are urgent if possible we encourage teams to try and work through the issue themselves with guidance from the UX champions.
+
+
+
+All new projects that have UI's and customer facing elements should consider using the design system within their project in order to make our products have a consistent look and feel. It also offers the benefits of familiarity to developers and confidence of the elements working together.
+
+
+
+## Angular
+
+
+
+Currently at the UKHO our first choice for language when building websites is Angular. This is where our currenty specialities lie but this could change and evolve overtime depending on current recruitment trends. 
```

Comment:
```suggestion
Currently at the UKHO our first choice for language when building websites is Angular. This is currently where our specialties lie but this could change and evolve overtime depending on current recruitment trends. 
```